services/121-service/src/migration/1739294207739-RenamePeopleAffected.ts

```diff
@@ -0,0 +1,17 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class RenamePeopleAffected1739294207739 implements MigrationInterface {
+  name = 'RenamePeopleAffected1739294207739';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `ALTER TABLE "121-service"."program_registration_attribute" ALTER COLUMN "export" SET DEFAULT '["all-registrations","included"]'`,

```

Comment:
Is this only changing what is set as the "default value"?

All existing rows with a value in the "export"-column need to be changes as well, right?
All existing "program_registration_attribute"-rows where a value exists of "all-people-affected" needs to be converted to "all-registrations".